docs/components/InputEmail/Web.stories.tsx

```diff
@@ -19,3 +22,233 @@ export const Basic = BasicTemplate.bind({});
+  const [description, setDescription] = useState<string>("");
+  const [error, setError] = useState<string>("");
+
+  const extraProps = {
+    invalid,
+    inline,
+    disabled,
+    description,
+    error,
+  };
+
+  const handleChange = (field: keyof typeof values) => (value: string) => {
+    setValues(prev => ({ ...prev, [field]: value }));
+  };
+
+  const renderBothVersions = (
+    title: string,
+    props: Record<string, unknown>,
+    field: keyof typeof values,
+  ) => (
+    <Grid>
+      <Grid.Cell size={{ xs: 12 }}>
+        <h3>{title}</h3>
+      </Grid.Cell>
+      <Grid.Cell size={{ xs: 6 }}>
+        <InputEmail
+          {...props}
+          version={1}
+          value={values[field]}
+          onChange={handleChange(field)}
+        />
+      </Grid.Cell>
+      <Grid.Cell size={{ xs: 6 }}>
+        <InputEmail
+          {...props}
+          version={2}
+          value={values[field]}
+          onChange={handleChange(field)}
+        />
+      </Grid.Cell>
+    </Grid>
+  );
+
+  return (
+    <Content>
+      <div style={{ display: "flex", flexDirection: "column", gap: "2rem" }}>
+        <Grid>
+          <Grid.Cell size={{ xs: 6 }}>
+            <h2>V1</h2>
+          </Grid.Cell>
+          <Grid.Cell size={{ xs: 6 }}>
+            <h2>V2</h2>
+          </Grid.Cell>
+        </Grid>
+
+        {renderBothVersions(
+          "Basic Input",
+          {
+            placeholder: "Enter your email",
+            ...extraProps,
+          },
+          "basic",
+        )}
+
+        {renderBothVersions(
+          "Right Aligned",
+          {
+            placeholder: "Right aligned",
+            align: "right",
+            ...extraProps,
+          },
+          "rightAlign",
+        )}
+
+        {renderBothVersions(
+          "Center Aligned",
+          {
+            placeholder: "Center aligned",
+            align: "center",
+            ...extraProps,
+          },
+          "centerAlign",
+        )}
+
+        {renderBothVersions(
+          "With Prefix",
+          {
+            placeholder: "Input with prefix",
+            prefix: { icon: "search" },
+            ...extraProps,
+          },
+          "prefix",
+        )}
+
+        {renderBothVersions(
+          "With Suffix",
+          {
+            placeholder: "Input with suffix",
+            suffix: { icon: "sparkles" },
+            ...extraProps,
+          },
+          "suffix",
+        )}
+
+        {renderBothVersions(
+          "With Prefix and Suffix",
+          {
+            placeholder: "Input with both",
+            prefix: { label: "Pre" },
+            suffix: { label: "Suf" },
+            ...extraProps,
+          },
+          "both",
+        )}
+
+        {renderBothVersions(
+          "Size small",
+          {
+            placeholder: "Small input",
+            size: "small",
+            ...extraProps,
+          },
+          "sizeSmall",
+        )}
+
+        {renderBothVersions(
```

Comment:
I think a [readonly](https://github.com/GetJobber/atlantis/blob/JOB-121642-do-it/docs/components/InputText/Web.stories.tsx#L110) variant would be useful like `InputText` has